src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -358,7 +401,7 @@ private static int BinarySearch(string concatenatedNames, ImmutableArray<Node> n
             out ImmutableArray<Node> sortedNodes)
         {
             // Generate index numbers from 0 to Count-1
-            var tmp = new int[unsortedNodes.Length];
+            int[]? tmp = new int[unsortedNodes.Length];
```

Comment:
wat?

Reply:
Needed to enable NRT (later tmp is assigned null)

Reply:
ah.  #resolved